resources/public/api/conf/3.0/schemas/retrieve/calculation/foreignPropertyIncome.json

```
@@ -1,13 +1,100 @@
 {
   "$schema": "http://json-schema.org/draft-04/schema#",
-  "description": "###TODO###",
-  "type": "object",
-  "properties": {
-    "###TODO###": {
-      "description": "Other properties",
-      "type": "string"
-    }
+  "description": "Array containing a breakdown of the income received for foreign property by country code for the retrieved tax calculation.",
+  "type": "array",
+  "items": {
```

Needs to be of type object otherwise properties aren't rendered:
```suggestion
  "items": {
    "type": "object",
```